zhaquirks/__init__.py

```diff
@@ -264,7 +267,7 @@ def handle_cluster_request(
         ] = None,
     ):
         """Handle the cluster command."""
-        if hdr.command_id == ZONE_STATE:
+        if hdr.command_id == ZONE_STATUS_CHANGE_COMMAND:
             if self._timer_handle:
```

Comment:
Can we at least add some log here?
```python
            _LOGGER.debug("ZONE_STATUS_CHANGE_COMMAND with args: %s", args)
```

Reply:
See https://github.com/zigpy/zha-device-handlers/pull/2231/commits/c16e581404eb1204d9ead1abe788ea5e10a06273
This prints something ``Motion detected: ZoneStatus.Alarm_1``.

I don't think we need to print the complete command for this:
```python
status_change_notification(zone_status=<ZoneStatus.Alarm_1: 1>, extended_status=<bitmap8.0: 0>, zone_id=0, delay=0)
```

Reply:
Actually, HA already logs something, so this seems to somewhat duplicate it:

```python
2023-02-27 00:04:53.504 DEBUG (MainThread) [zigpy.zcl] [0xF31E:1:0x0500] <IEEE> - Motion detected: ZoneStatus.Alarm_1
2023-02-27 00:04:53.505 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0xF31E:1:0x0500]: Updated alarm state: ZoneStatus.Alarm_1
```

Reply:
It could be interesting to know about other values but inside of `if hdr.command_id == ZONE_STATUS_CHANGE_COMMAND and args[0] & 3:` we can't see it.
I would suggest move it outside the 'if' or remove it.
I haven't checked if ZHA would log other values than alarm1 & 2.

Reply:
ZHA should log everything all possible alarm states: https://github.com/home-assistant/core/blob/0f018665080c6eaf53153b5794fad5e8b9d4a046/homeassistant/components/zha/core/channels/security.py#L344-L349

Reply:
Thanks for checking (I'm on the phone now)
Then I believe that can be removed.

Reply:
Hmm, never mind. ZHA does *not* log all possible alarm states (sets `state = args[0] & 3` first and then only prints `state`).
That code seems to somewhat override the other bits then... But I guess it's fine because ZHA doesn't do anything with the other bits (tamper, battery, ...)?

Reply:
Implanting sensor for tamper is asked for but its out of scope for this issue.

Reply:
I've removed the log message for now. It would only log for sensors that use this quirk.
If other bits of that `zone_status` are needed, ZHA (HA Core) should log it.